src/ElmSharp/ElmSharp/PreloadedWindow.cs

```diff
@@ -69,6 +96,21 @@ public void WarmupWidgets()
             //TODO: Consider to call Image.LoadAsync()
         }
 
+        public void WarmupWearableWidgets()
+        {
+            try
+            {
+                Assembly assem = Assembly.Load("ElmSharp.Wearable");
+                var type = assem.GetType("ElmSharp.Wearable.Preload");
+                type.GetMethod("WarmupWidgets", BindingFlags.Public | BindingFlags.Static).Invoke(null, new object[] { this });
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.ToString());
```

Comment:
Catching all Exception results in the Console output only and the program is alive, no matter what the exception occurs. Is that what you intended?
I understand that there are no instances where exceptions are not forced to occur at the API level unless they are special.

Reply:
Yes. This reflection call will be invoked by a candidate process, and it calls preload feature in `ElmSharp.Wearable` which is technically not mandatory for launching an application.
An application should not be affected even if this call fails. At the same time, I thought it would be better to show the failure reason on console log.